### ISSUE TITLE
Running new VkDescriptorBufferInfo() in an electron render process causes crash on getArrayBufferFromAddress() in VkBuffer initialisation.

### ISSUE THREAD

**Issue by Someoneamzing** [maierfelix/nvk#50]
Running on Windows 10 64x AMD Processor. Don't have access to other platforms so unsure if they are affected.

Versions:
electron@8.0.0
node@12.13.0
nvk@0.7.8

Attempting to create a new VkDescriptorBufferInfo() in an electron render process causes the process to crash. After debugging with a ton of console.logs in interfaces.js the program stops after the call to getArrayBufferFromAddress in the VkBuffer initialisation function setting this.memoryBuffer. (Position [line 5: col 36777] in the unmodified interfaces.js) 

Below is a snippet of the function.
```
function VkBuffer(e){if(this.memoryBuffer=null,this.memoryAddress=BI0,this.memoryOffset=0,void 0!==e){if(void 0===e.$memoryBuffer)throw new Error("'VkBuffer' doesn't take any arguments");if(void 0!==e.$memoryOffset){let t=getAddressFromArrayBuffer(e.$memoryBuffer);this.memoryBuffer=getArrayBufferFromAddress(t+BigInt(e.$memoryOffset),BI8),this.memoryOffset=e.$memoryOffset}else this.memoryBuffer=e.$memoryBuffer}else this.memoryBuffer=new ArrayBuffer(8);this.memoryAddress=getAddressFromArrayBuffer(this.memoryBuffer),this.memoryView=new DataView(this.memoryBuffer,0,8)}
```

The function never returns and the DevTools for the render process becomes disconnected and the page that was loaded while running the script disappears.

Running the same file through node directly has no problem and the script performs as expected.

The memory addresses and sizes passed to getArrayBufferFromAddress seem fine and other calls to the same function do not cause this issue.

This only happens for VkDescriptorBufferInfo() as far as i am aware. Other calls to VkBuffer() have no issues and no matter where in the application I create a VkDescriptorBufferInfo() it crashes, so it isn't a case of too little memory. Other calls to the nvk.node files and getArrayBufferFromAddress() work fine so it isn't a case of incorrectly built files or missing bindings (electron needs some binaries to be rebuilt for it)

I have attached a project with the node and electron tests as well as the modified interfaces.js with the tracking console.logs. No other modifications have been added and as I said the same modified files run fine in a regular node process.

The project is a fully functional lighting calculator for 2d scenes so there is a fair bit of Vulkan boilerplate code. engine_launcher.js is the script that passes the necessary data to the vulkan app to calculate lighting for a scene. This scene is an empty scene with a single light. and the result is output as ascii art in the console. LightingEngine.js is the actual Vulkan app that implements the compute shader dispatching and result fetching.

I am posting this issue here as I don't have the experience or know-how to debug any further than the call to the .node binaries.

Project Setup:
1. unzip project contents into a folder
2. cd into the created directory.
3. `npm install`
4. `replace ".\interfaces.js" ".\node_modules\nvk\generated\1.1.126\win32"` Adds the debug console.logs to the affected calls.

Running:
```
//Electron (Crashes)
npm run
//Regular Node (Successfully computes lighting data and outputs it)
node engine_launcher.js
```

Here is the project: [nvk-error-tests.zip](https://github.com/maierfelix/nvk/files/4168895/nvk-error-tests.zip)

**Comment by maierfelix** [comment]
Hey, thanks for reporting -

I pushed a fix to master and I managed to run your project without crashing: https://prnt.sc/qz00gg
Please pull the changes and validate that this fixes your problem

**Comment by Someoneamzing** [comment]
Fixes the issue perfectly, thanks.

**Comment by maierfelix** [comment]
Should be fixed in `v0.7.9`
